docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       - PERSISTENCE=1
       - LOCALSTACK_AUTH_TOKEN=${LOCALSTACK_AUTH_TOKEN}
       - SNAPSHOT_LOAD_STRATEGY=ON_STARTUP
+      - SNAPSHOT_FLUSH_INTERVAL=60
```

Comment:
If our load strategy has been set to load the state on_startup (default is on_request), do we actually need to worry about keeping the state up to date so frequently? Could we flush on shutdown, which would still speed up the start time? [source](https://docs.localstack.cloud/user-guide/state-management/persistence/#save-strategies)

Reply:
In a perfect world a Shutdown flush would be ideal.  I've seen localstack run itself out of memory after running for too long which would make the Shutdown Flush fail.  And if you get too Ctrl-C happy at the end of the day when wanting to shut it down you could also inadvertently stop the flush.
An option would be to include these options as env vars and have each dev set their preferences?

Reply:
I'm cool with sticking to the options we've set. Let's give them a shot until they start causing problems. My thinking was, since this isn't something we absolutely need to be super reliable, we could ease up on the expectations. No big deal. Thanks for fine tuning it!